a8cteam51/build-processes-demo
# Add sass build scripts for theme scss->css

#### Changes proposed in this Pull Request

- Added `sass` to devDependencies
- Added `build:theme-css` `build:theme-editor-css` `start:theme` scripts


#### Testing instructions

1. Run `composer update`
2. Add new files to the `assets/core folder` and include them in `style.scss` or `style-editor.scss` files and then run the new commands
3. You should have an updated `theme/style.css` and `theme/style-editor.css` file respectively

#### Discussion

- This is the first basic step just to see if this will be the way forward
- I didn't include any additional plugins (postcss, autofixer, etc.)
- Mu-plugins SCSS->CSS is built with wp-scripts so I didn't create a build process for that because it already exists
